frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -645,12 +652,19 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
         )}
         isRequired
       >
-        <TextInput
-          onChange={setBsName}
-          value={bsName}
-          placeholder="my-backingstore"
-          aria-label={t('noobaa-storage-plugin~Backing Store Name')}
-        />
+        <Tooltip
+          content="Max of 43 characters are allowed"
```

Comment:
```suggestion
          content="Max. of 43 characters are allowed"
```
@yuvalgalanti Max. or Maximum, wdyt?

Reply:
Maximum

Reply:
@bipuladh , suggested change done...

Reply:
Let me know if it should be changed to `Maximum` ...

Reply:
I have updated the line to the one which @yuvalgalanti has suggested...